crates/astria-composer/src/searcher/executor.rs

```diff
@@ -271,27 +403,30 @@ impl SequencerClient {
     }
 
     /// Wrapper around [`Client::get_latest_nonce`] with a 1s timeout.
-    async fn get_latest_nonce(&self, address: Address) -> eyre::Result<NonceResponse> {
+    async fn get_latest_nonce(
+        &self,
+        address: Address,
+    ) -> Result<NonceResponse, SequencerClientError> {
         tokio::time::timeout(
             Duration::from_secs(1),
             self.inner.get_latest_nonce(address.0),
         )
         .await
-        .wrap_err("request timed out")?
-        .wrap_err("RPC returned with error")
+        .map_err(|_e| SequencerClientError::Timeout)?
```

Comment:
Don't ignore the error. I know it doesn't add a lot of info, but it's easier to understand if you don't.